docs/user/02-configuration.md

```diff
@@ -73,15 +73,16 @@ Use the following sample CRs as guidance. Each can be applied immediately when y
 | **specHash** (required)                              | integer    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
-| **state** (required)                                 | string     | Can have one of the following values: `Ready`, `Error`, `Processing`, `Warning`. `Ready` state is set when all the resources are deployed successfully and backend is connected. It gets `Warning` state in case backend is not specified, NATS module is not installed or EventMesh secret is missing in the cluster. `Error` state is set when there is an error. `Processing` state is set if recources are being created or changed. See the next section for more details. |
+| **state** (required)                                 | string     | Can have one of the following values: `Ready`, `Error`, `Processing`, `Warning`. `Ready` state is set when all the resources are deployed successfully and backend is connected. It gets `Warning` state if backend is not specified, NATS module is not installed or EventMesh secret is missing in the cluster. `Error` state is set when there is an error not caused by a user. `Processing` state is set when eventing is initiliazed or backend is being switched. See the next section for more details. |
 
 <!-- TABLE-END -->
 
 <h3>More Details About Eventing CR State</h3>
-The following table provides more details on the overall state of the Eventing CR:
-- The <b>State</b> column has the overall state of the Eventing CR.
+The following table provides more details on the overall state of the Eventing CR: 
+
 - The <b>Backend Config</b> column is part of the eventing CR that is either available or mistakenly not specified by a user.
-- The <b>Backend State</b> column describes the state of the NATS backend or EventMesh Secret existence. 
+- The <b>Backend State</b> column describes the state of the NATS backend or EventMesh Secret existence.
+- The <b>Eventing State</b> column has the overall state of the Eventing CR.
 
 Warnings indicate that user action is required, that is, the user must install the NATS module or create an EventMesh Secret.
 
```

Comment:
In general, how about the following order of rows and columns:

```suggestion

| Backend Type   | Backend Config | Backend State                                                                                                                               | Eventing State | Impact on Event Flow                     |
|----------------|----------------|---------------------------------------------------------------------------------------------------------------------------------------------|----------------|------------------------------------------|
| NATS/EventMesh | missing        | n/a                                                                                                                                         | Warning        | No events will be accepted or dispatched |
| NATS/EventMesh | exists         | Processing (mainly happens during initialization and backend switching)                                                                     | Processing     | No events will be accepted or dispatched |
| NATS           | exists         | Warning (NATS deletion blocked)                                                                                                             | Ready          | Events will be accepted or dispatched    |
| NATS           | exists         | Error (NATS ist unavailable or cannot be connected)                                                                                         | Warning        | No events will be accepted or dispatched |
| NATS           | exists         | Missing                                                                                                                                     | Warning        | No events will be accepted or dispatched |
| EventMesh      | exists         | Error (secret for EventMesh missing)                                                                                                        | Warning        | No events will be accepted or dispatched |
| NATS/EventMesh | exists         | Error (cases not caused by a user, such as cannot create EPP deployment or cannot start subscription manager although backend is available) | Error          | No events will be accepted or dispatched |
```

Reply:
We discussed the order with Korbinian and we agreed to this order. Why do you want to change it?

Reply:
Because I think a) backend missing or existing is the first important distinction; with a missing backend, all later steps are irrelevant. That's why I pulled that row up.

Furthermore, because as a user, I think it's easiest to know whether I have NATS or EventMesh, so I pulled that column to the first position. 
This also matches the fact that this is about the impact of the backend to the Eventing CR State and Event Flow; so the focus is on the backend. Conversely, if the purpose of the table had been "Understanding the causes of the Eventing state"; I'd pull the "Eventing State" column to the first position.

In general, I personally find option A easier to understand than option B:

Option A
<img width="1093" alt="image" src="https://github.com/kyma-project/eventing-manager/assets/76950046/5f0f252a-b9c4-4f5c-9489-307dba31ebac">

Option B
<img width="1019" alt="image" src="https://github.com/kyma-project/eventing-manager/assets/76950046/6d1eb9a8-fdce-40d2-a2eb-17a16a0b4ef5">